openfisca_france_dotations_locales/variables/dotation_solidarite_rurale_fractions/cible.py

```diff
@@ -0,0 +1,80 @@
+import pandas as pd
+
+from openfisca_core.model_api import *
+from openfisca_france_dotations_locales.entities import *
+
+
+class indice_synthetique_dsr_cible(Variable):
```

Comment:
Conserver la même règle de nommage et commencer par la dotation concernée ?
```suggestion
class dsr_cible_indice_synthetique(Variable):
```